src/dbnode/generated/proto/namespace/namespace.proto

```diff
@@ -33,6 +33,31 @@ message NamespaceOptions {
     bool coldWritesEnabled                          = 10;
     NamespaceRuntimeOptions runtimeOptions          = 11;
     google.protobuf.BoolValue cacheBlocksOnRetrieve = 12;
+    AggregationOptions aggregationOptions           = 13;
+}
+
+message AggregationOptions {
+    repeated Aggregation aggregations = 1;
+}
+
+message Aggregation {
+    AggregationAttributes attributes    = 1;
+    DownsampleOptions downsampleOptions = 2;
+}
+
+message AggregationAttributes {
+    int64 resolutionNanos = 1;
+    AggregationType type  = 2;
+}
+
+message DownsampleOptions {
+    bool all = 1;
+}
+
+enum AggregationType {
```

Comment:
can a simple `aggregated` boolean suffice?

Reply:
if `UNAGGREGATED` is not included, are the raw metrics dropped?

Reply:
Not by default. Raw metrics are dropped if `DownsampleOptions.All = false`